LayoutTests/platform/ios/TestExpectations

```diff
@@ -3932,6 +3932,10 @@ imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-allow
 imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-allowed-for-some.https.sub.html [ DumpJSConsoleLogInStdErr Failure Pass ]
 imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-disallowed-for-all.https.sub.html [ DumpJSConsoleLogInStdErr Failure Pass ]
 
+# Close Watcher tests rely on sending escape keys from test_driver which doesn't work on iOS webkit.org webkit.org/b/274832
```

Comment:
webkit.org appears twice (we typically put the bug number in front of the test paths too)